app/assets/stylesheets/components/_footer.scss

```diff
@@ -33,6 +33,10 @@ footer#site-footer {
     margin: 1rem 0;
   }
 
+  .site-mode-link-container {
+    text-align: center;
+  }
```

Comment:
Can you upload a screenshot of what this looks like, please?

(Since we can't see Lite Mode on a PR review app)

Reply:
I can't get lite mode to show up locally now. I'll see if I can get it working locally and do a grab. That CSS change is for the full site footer link.

Reply:
Oh. Don't worry about it then.